deploy/helm/jumpstarter/values.yaml

```diff
@@ -66,6 +67,15 @@ jumpstarter-controller:
 
     namespace: ""
 
+    grpcConfig: |
+      apiVersion: jumpstarter.dev/v1alpha1
+      kind: GrpcConfiguration
+      keepalive:
+        # Safety: potentially makes server vulnerable to DDoS
+        # https://grpc.io/docs/guides/keepalive/#how-configuring-keepalive-affects-a-call
+        minTime: 3s
+        permitWithoutStream: true
+
```

Comment:
Actually the AuthenticationConfiguration is something that we need to break down, it makes helm usage hard, we should individualize the parameters and not add extra config formats.

```suggestion
    grpcConfig:
      keepalive:
        minTime: 3s
        permitWithoutStream: true
```